README.md

```diff
@@ -98,15 +96,12 @@ The integration test executable is located in the following path:
 
 1. Tests run
 
-    See "Step 4: deploy" above.
+    See "Step 4: Deploy" above.
 
-1. Known issues
+2. Known issues
```

Comment:
this should stay "1.". Markdown viewers automatically re-number the sections as needed (look at the front page, where you see the readme.md file rendered - it is correct.)